### PR TITLE
add all images to the engine

### PR DESCRIPTION
**Pull request by Norrox**
image

### CODE REVIEW

**Comment by sshiiden** [comment]
image
The text doesn't show, maybe becouse we didn't put the font in `thirdparty/fonts`

**Comment by Norrox** [comment]
> image The text doesn't show, maybe because we didn't put the font in `thirdparty/fonts`

what font are we using 
image

**Comment by sshiiden** [comment]
I just finished compiling with the font and the result is the same.
Only solution is to change all svg text to paths, I will do that then push it to blazium-assets.

**Comment by Norrox** [comment]
> I just finished compiling with the font and the result is the same. Only solution is to change all svg text to paths, I will do that then push it to blazium-assets.

add the font we are using as well! :)

**Review by RandomOfNoWhere — APPROVED** [review]

**Comment by sshiiden** [comment]
Updated Drag-And-Drop Assets PR
https://github.com/blazium-engine/blazium-assets/pull/23

**Comment by Norrox** [comment]
> Updated Drag-And-Drop Assets PR [blazium-engine/blazium-assets#23](https://github.com/blazium-engine/blazium-assets/pull/23)

used your updated assets
https://github.com/blazium-engine/blazium/pull/42/commits/e95975cc41be185404ee3928fe3c84d45ffb9c41

File: icon_outlined.png
Comment (jss2a98aj) [inline icon_outlined.png:1]: Is there a reason this icon changed resoloution? Was the previous version incorrectly swapped?

File: platform/android/java/lib/res/mipmap/icon_monochrome.png
Comment (jss2a98aj) [inline platform/android/java/lib/res/mipmap/icon_monochrome.png:1]: The monochrome icon is only used in Godot 4.4 and can be removed.

**Review by sshiiden — APPROVED** [review]

File: platform/windows/godot_console.ico
Comment (jss2a98aj) [inline platform/windows/godot_console.ico:1]: The 16x16 version of this icon appears to incorrectly be 16x17, and should probably be fixed.

File: icon_outlined.png
Comment (Norrox) [reply icon_outlined.png:1]: hmm original are 248x235

File: platform/windows/godot_console.ico
Comment (sshiiden) [reply platform/windows/godot_console.ico:1]: https://github.com/blazium-engine/blazium-assets/pull/23/commits/469d8338788e6cb220a6b1b6b371f29214c99c62
It now it should be 16x16

File: icon_outlined.png
Comment (jss2a98aj) [reply icon_outlined.png:1]: Looking at the original, I think it was that size to avoid having unused space.

**Review by jss2a98aj — APPROVED** [review]

**Review by Bioblaze — APPROVED** [review]

**Review by flare199 — APPROVED** [review]

**Review by albertoacv4498 — APPROVED** [review]
All images seem correct to me. Nothing extra to comment about.
Wawa c:

**Pull request merged**